auth0-js/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Adrian Chia <https://github.com/adrianchia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace auth0 {
+declare module auth0 {
```

Comment:
Keep it as `namespace`. There is no functional difference between `declare module Identifier` and `declare namespace Identifier` and the latter is preferred as a convention.